packages/terra-divider/src/Divider.module.scss

```diff
@@ -1,12 +1,33 @@
 :local {
   .divider {
-    border-bottom-width: 0;
-    border-color: var(--terra-divider-border-color, #dedfe0);
-    border-left-width: 0;
-    border-right-width: 0;
-    border-style: solid;
-    border-top-width: var(--terra-divider-border-top-width, 1px);
-    height: 0;
-    margin: 0;
+    border: 0;
+    height: 1.5em;
```

Comment:
Adding height and line-height styles make this version of the divider render more space vertically when there is no text compared to the version that is in master. 
### Master
<img width="161" alt="Screen Shot 2019-03-25 at 6 35 17 PM" src="https://user-images.githubusercontent.com/633148/54960951-d445a480-4f2c-11e9-9cbc-2b850f1f6963.png">

### PR
<img width="147" alt="Screen Shot 2019-03-25 at 6 35 30 PM" src="https://user-images.githubusercontent.com/633148/54960952-d445a480-4f2c-11e9-8986-a4beab415905.png">

Need to sync up with @neilpfeiffer and see if this is an intentional height increase. If so, I'd consider this a non-passive change as it will impact spacing where prior to this change, the divider only rendered at 1px height.

Reply:
I put in some changes to hopefully correct this a little bit: [f614149](https://github.com/cerner/terra-core/pull/2296/commits/f6141490bc5feede0c8c51b8f9513ce833957c3d). The custom text styling is now conditional, so the changes to the standard divider should hopefully be more passive now.